app/bundles/CampaignBundle/Views/Campaign/leads.html.twig

```diff
@@ -0,0 +1,92 @@
+{{ dump('Campaign/leads') }}
+<div class="pa-md bg-auto">
+    {% if items|length > 0 %}
+    <div class="shuffle-grid row">
+        {% for item in items %}
+            <div class="shuffle shuffle-item card grid col-sm-6 col-lg-4">
+                <div class="panel ovf-h" style="border-top: 3px solid {{ item['color'] }};">
+                    <div class="box-layout">
+                        <div class="col-xs-4 va-m">
+                            <div class="panel-body">
+                                <span class="img-wrapper img-rounded" style="width:100%">
+                                    {% set preferred = item['preferred_profile_imate']|default(false) %}
+                                    {% if 'gravatar' == preferred or false == preferred %}
+                                        {# I assume this works correctly #}
+                                        <?php $img = $view['gravatar']->getImage($item['email'], '250'); ?>
+                                        {% set img = gravatarGetImage(item['email'], '250') %}
+                                    {% else %}
+                                        {# @todo #}
+                                        <?php $socialData = \Mautic\CoreBundle\Helper\Serializer::decode($item['social_cache']); ?>
```

Comment:
@todo Is there a twig extension that can decode this?

Reply:
I don't see this template in the diff anymore. Please create a new extension if there is none for this yet.

Reply:
@escopecz I cannot see where the "leads.html.php" template is being used so I just deleted the twig template.

Reply:
Interesting. I don't see it used either. If the "Contacts" tab on the campaign detail page still works then it looks like we can remove it.